test/unit/reducers/projects.js

```diff
@@ -45,7 +47,7 @@ test('projectCreated', (t) => {
 
 test('updateProjectSource', reducerTest(
   reducer,
-  initProjects({[projectKey]: false}),
+  initProjects({[projectKey]: true}),
```

Comment:
The change you commented on below broke this particular test; changing `false` to `true` fixes it without affecting the purpose of the test itself.